<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>584</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>80</y>
      <width>581</width>
      <height>301</height>
     </rect>
    </property>
    <row>
     <property name="text">
      <string>r1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>r2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>r3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>r4</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>c1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>c2</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>c3</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>c4</string>
     </property>
    </column>
    <item row="1" column="0">
     <property name="text">
      <string>fsfs</string>
     </property>
    </item>
    <item row="1" column="1">
     <property name="text">
      <string>sfsf</string>
     </property>
    </item>
    <item row="1" column="2">
     <property name="text">
      <string>sfsf</string>
     </property>
    </item>
    <item row="1" column="3">
     <property name="text">
      <string>sfsf</string>
     </property>
    </item>
    <item row="2" column="0">
     <property name="text">
      <string>kkk</string>
     </property>
    </item>
    <item row="2" column="1">
     <property name="text">
      <string>sdjisj</string>
     </property>
    </item>
    <item row="2" column="2">
     <property name="text">
      <string>kkjj</string>
     </property>
    </item>
    <item row="2" column="3">
     <property name="text">
      <string>hhhhh</string>
     </property>
    </item>
    <item row="3" column="0">
     <property name="text">
      <string>uuut</string>
     </property>
    </item>
    <item row="3" column="1">
     <property name="text">
      <string>tkkkk</string>
     </property>
    </item>
    <item row="3" column="2">
     <property name="text">
      <string>kkkk</string>
     </property>
    </item>
    <item row="3" column="3">
     <property name="text">
      <string>iii</string>
     </property>
    </item>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>310</y>
      <width>118</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>420</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>PushButton</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>949</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
